backoffice/src/Models/Entities/WreckMaterial.cs

```diff
@@ -11,6 +11,7 @@ public class WreckMaterial
 
     public virtual Droit? Droit { get; set; }
     public string Name { get; set; } = string.Empty;
+    public Address Address { get; set; } = new();
```

Comment:
Can we rename this StorageAddress pls?
public Address StorageAddress { get; set; } = new();